client/SymbolClient.py

```diff
@@ -54,8 +54,36 @@ def __init__(self, address):
 		self.voting_public_keys = []
 
 
-class SymbolPeerClient:
+class SymbolLightRESTClient:
```

Comment:
Should light rest have other methods also, like ``get_chain_height``?

Reply:
Yes, but we do need it, as chain height we can be queried from peer.